From: Chris Samuel <chris@csamuel.org>
Date: Wed, 30 Nov 2011 22:28:37 +1100
Patch-mainline: no
References: FATE#306586
Subject: [PATCH] Prefix mount messages with btrfs: for clarity

Currently when mounting a btrfs filesystem a user searching dmesg
has no obvious string to search for as currently we report something
cryptic like:

[ 5775.216078] device label DR devid 1 transid 15757 /dev/sdh1

It would be much nicer if there was some mention of btrfs to find
(as other filesystems generally provide), so this change prefixes
that message with btrfs: to make it appear as:

[ 5775.216078] btrfs: device label DR devid 1 transid 15757 /dev/sdh1

Signed-off-by: Chris Samuel <chris@csamuel.org>
Signed-off-by: David Sterba <dsterba@suse.cz>
---
 fs/btrfs/volumes.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/fs/btrfs/volumes.c
+++ b/fs/btrfs/volumes.c
@@ -866,9 +866,9 @@ int btrfs_scan_one_device(const char *pa
 	if (disk_super->label[0]) {
 		if (disk_super->label[BTRFS_LABEL_SIZE - 1])
 			disk_super->label[BTRFS_LABEL_SIZE - 1] = '\0';
-		printk(KERN_INFO "device label %s ", disk_super->label);
+		printk(KERN_INFO "btrfs: device label %s ", disk_super->label);
 	} else {
-		printk(KERN_INFO "device fsid %pU ", disk_super->fsid);
+		printk(KERN_INFO "btrfs: device fsid %pU ", disk_super->fsid);
 	}
 
 	printk(KERN_CONT "devid %llu transid %llu %s\n",
